studio/components/interfaces/Auth/RedirectUrls/RedirectUrls.tsx

```diff
@@ -49,19 +55,25 @@ const RedirectUrls = () => {
       return
     }
 
-    const { error } = await authConfig.update({ URI_ALLOW_LIST: payloadString })
-    if (!error) {
-      setOpen(false)
-      ui.setNotification({ category: 'success', message: 'Successfully added URL' })
-    } else {
-      ui.setNotification({
-        error,
-        category: 'error',
-        message: `Failed to update URL: ${error?.message}`,
-      })
-    }
-
-    setSubmitting(false)
+    updateAuthConfig(
+      { projectRef: projectRef!, config: { URI_ALLOW_LIST: payloadString } },
+      {
+        onError: (error) => {
```

Comment:
likewise as per https://github.com/supabase/supabase/pull/16227/files#r1289813556
shift all these to the mutation declaration

use isLoading from the mutation rather than isSubmitting from the Form

Reply:
Can't, due to `resetForm`.  I've removed `isSubmitting`.